ert/storage/_storage.py

```diff
@@ -396,52 +396,42 @@ def _add_numerical_data(
         raise ert.exceptions.StorageError(meta_response.text)
 
 
+# the local variable dataframe is not read by pylint as pandas.Dataframe,
+# but due to chunking a pandas.TextFileReader
+# https://stackoverflow.com/questions/41844485/why-the-object-which-i-read-a-csv-file-using-pandas-from-is-textfilereader-obj
 def _response2records(
     response_content: bytes, metadata: _NumericalMetaData
 ) -> ert.data.RecordCollection:
-
+    assert metadata
```

Comment:
Why is this needed?

Reply:
I don't believe it is needed but decided to keep it, with the idea that whoever set it had a reason to do so.

Reply:
As far as I can tell you added this line. There was no naked assert on the metadata before, so you are the one that needs to have a reason to do so :stuck_out_tongue_winking_eye: